src/pages/household/input/VariableEditor.jsx

```diff
@@ -176,7 +174,7 @@ function HouseholdVariableEntityInput(props) {
         (householdId) => {
           let newSearch = new URLSearchParams(window.location.search);
           newSearch.set("household", householdId);
-          setSearchParams(newSearch);
+          window.history.replaceState({}, '', `${window.location.pathname}?${newSearch}`);
```

Comment:
Is there a reason for this? My understanding was that using `setSearchParams` ensured that React triggered all the processes needed to respond to a change in state. Does that still happen here?